modules/Microsoft.Network/connections/deploy.bicep

```diff
@@ -89,6 +89,9 @@ param virtualNetworkGateway2 object = {}
 @description('Optional. The remote peer. Used for connection type [ExpressRoute].')
 param peer object = {}
 
+@description('Optional. The Authorization Key to connect to an Express Route Circuit. Used for connection type [ExpressRoute].')
```

Comment:
Is this key a secret? If so, we should add the `@secure` decorator, if not, please disregard.